testsuite/features/support/env.rb

```diff
@@ -18,9 +18,9 @@
 
 server = ENV['SERVER']
 $debug_mode = true if ENV['DEBUG']
```

Comment:
we should probably use same startegy for `DEBUG` variable too then

(btw do we use that variable at all? if yes, is it documented?)

Reply:
I agree, but I could not find where we set `DEBUG`: it's not set by terracumber.
I prefer to keep the scope of this PR as it is: fix the vars that we set with terracumber, and we fail to check correctly.